index.js

```diff
@@ -8,7 +8,15 @@ import { PingClient } from './lib/ping.js';
 
 import { ResponseCache } from './lib/response-cache.js';
 
-import { GamificationFallbackPlugin } from './plugins/gamification-fallback.js';
+// REMOVED: due to GamificationFallbackPlugin importing the whole gamification bundle
+// import { GamificationFallbackPlugin } from './plugins/gamification-fallback.js';
+
+// IDEA: create fake instance to throw error anywhere we are importing GamificationFallbackPlugin
```

Comment:
Makes sense. Here's where it's being used:

https://github.com/KanoComputing/kit-app-ui/blob/master/lib/models/gamification-client.js#L1

It might be worth bumping the major version as well.